app/models/location.rb

```diff
@@ -40,36 +40,36 @@ class Location < ApplicationRecord
 
   geocoded_by :address
 
-  attr_accessor :country
-
   def address
-    [address_line1, zip_code, city, state,
-     ISO3166::Country.find_country_by_alpha2(country_code)].compact.join(', ')
+    [
+      address_line1, zip_code, city, state,
+      ISO3166::Country.find_country_by_alpha2(country_code)&.unofficial_names&.first
+    ].compact.join(', ')
   end
 
   def country
-    ISO3166::Country.find_country_by_alpha2(country_code).unofficial_names[0]
+    ISO3166::Country.find_country_by_alpha2(country_code)&.unofficial_names&.first
   end
 
   def convert_country_to_country_code
-    if country
-      country_alpha2 = ISO3166::Country.find_country_by_name(country)
-      country_code = country_alpha2.alpha2 if country_alpha2
-    end
+    return unless country
+
+    country_alpha2 = ISO3166::Country.find_country_by_name(country)
+    self.country_code = country_alpha2.alpha2 if country_alpha2
   end
 
   def country_code_to_upcase
     country_code.upcase!
   end
 
   def country_exists?
```

Comment:
Given the naming of the method, I would expect a boolean as the returned field (true / false)

Reply:
This method is used as a validation (see line 34). Would you just remove the question mark?

Reply:
Yes, it's a detail, but still :D 

https://guides.rubyonrails.org/v3.2/active_record_validations_callbacks.html#custom-methods